src/BaseTemplates/WebAPI/project.json

```diff
@@ -41,7 +41,8 @@ $endif$
   "content": [
     "wwwroot",
     "Views",
-    "appsettings.json"
+    "appsettings.json",
+    "web.config"
```

Comment:
For the project.json changes, you also need to update all the project.json files under src\Rules\**\project.json